### ISSUE TITLE
How exactly I'm supposed to run this?

### ISSUE THREAD

**Issue by faaiz99** [LinWeizheDragon/AutoFidgetDetection#3]
Can you provide a sample of interview_questionnaire_responses.xlsx

**Comment by LinWeizheDragon** [comment]
You don't need this file to run fidgeting detection using the code. Since the data release was not approved due to privacy and ethical agreements, we are not able to release the data for training and testing. You can follow the code and the paper to reproduce training on your own dataset.

**Comment by faaiz99** [comment]
I don't have my own dataset. Can the pre-trained model be run? without acquiring any dataset?

**Comment by LinWeizheDragon** [comment]
Yes, I think the pre-trained model can be run without your own dataset. Just follow the instruction to download a demo video (which is from a public dataset) and replace it with your own video.

**Comment by faaiz99** [comment]
I need to demonstrate the working of auto fidget detection. Would the pre-trained model be able to do that? Exactly like you have mentioned in the paper.

**Comment by LinWeizheDragon** [comment]
Unfortunately, I have left the group for a few years and I don't have permission to release either raw data or preprocessed features of participants (which are also private according to participant agreement).
The pre-trained model is the fidgeting detector that is used in the paper. You can replace the demo video with your own video and extract fidgeting features. If my memory serves me right, these features are 0/1 features on each frame. You can leverage these features in your own classification stage (e.g. a time-aware neural classifier or something else).
